engine/src/render/mesh/plugin.cpp

```diff
@@ -117,7 +117,8 @@ void cubos::engine::renderMeshPlugin(Cubos& cubos)
                 }
                 else if (entry.firstBucketId != RenderMeshPool::BucketId::Invalid)
                 {
-                    cmds.add(ent, RenderMesh{.firstBucketId = entry.firstBucketId}).remove<LoadRenderVoxels>(ent);
+                    auto assetRead = assets.read(entry.asset);
+                    cmds.add(ent, RenderMesh{.firstBucketId = entry.firstBucketId, .baseOffset = assetRead->size() / 2U}).remove<LoadRenderVoxels>(ent);
```

Comment:
Shouldn't the offset be the symetric/negative of this? From the sample scenes, it looks like it should